plugins/webhook/src/main.ts

```diff
@@ -212,6 +219,10 @@ class WebhookPlugin extends ScryptedDeviceBase implements Settings, MixinProvide
     }
 
     async getMixin(mixinDevice: any, mixinDeviceInterfaces: ScryptedInterface[], mixinDeviceState: { [key: string]: any; }): Promise<any> {
+        const id = mixinDeviceState.id;
+        if (this.createdMixins.has(id)) {
+            return this.createdMixins.get(id);
```

Comment:
this is incorrect, the underlying mixinDevice may have changed due to plugin reload or device reload. should return a new instance.